### PR TITLE
add activity report

### PR DESCRIPTION
**Pull request by francisco-core**

### CODE REVIEW

File: content/about/about-us.en.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ can become even more active in the defense of privacy by [becoming a
 member](/community/become-a-member)! 
 
 
-#### Financial Transparency Reports
+#### Reports
 
+##### 2020
+
+* [Activity report 2018-2020](/documents/activities/relatorio-atividades-2020.pdf) (in Portuguese)
```
Comment (anadahz) [inline content/about/about-us.en.md:31]: The file in `/documents/activities/relatorio-atividades-2020.pdf` doesn't exist.

Minor: `Activity report` should be `Activity Report` or all other items in the list should be in lowercase.

File: content/about/about-us.pt.md [hunk 2]
```diff
@@ -16,10 +16,14 @@ privacidade e segurança digital como direitos fundamentais para uma
...
 
  Já és frequente nos nossos eventos, fórum ou chat? Sabe como podes tornar-te ainda mais ativo(a) na defesa da privacidade, [torna-te membro](/community/become-a-member)!
  
-#### Relatórios Financeiros de Transparência
 
+#### Relatórios
+
+##### 2020
+
+* [Relatório de Atividades 2018-2020](/documents/activities/relatorio-atividades-2020.pdf)
```
Comment (anadahz) [inline content/about/about-us.pt.md:28]: Same as in the English version.
The file in `/documents/activities/relatorio-atividades-2020.pdf` doesn't exist.

**Review by anadahz — CHANGES_REQUESTED** [review]
@francisco-core Thank you very much for this PR.

Please, add the missing file and keep the case of the items in the list consistent.

File: content/about/about-us.pt.md (hunk 2 above)
Comment (francisco-core) [reply content/about/about-us.pt.md:28]: Well spotted. Just uploaded it. Please check again.

File: content/about/about-us.en.md (hunk 1 above)
Comment (francisco-core) [reply content/about/about-us.en.md]: fixed

**Comment by francisco-core** [comment]
Both addressed @anadahz. Merging.

**Comment by anadahz** [comment]
Thanks @francisco-core .

**Pull request merged**